DataDog/dd-trace-java
# jmxfetch wildfly crasher fix

* upgrade jmxfetch to latest version
* add logger instrumentation to make java.util.logging.Logger safe when used by jmxfetch

Still working on:
* add safetly checks to logger instrumentation to ensure private constructor is present before applying instrumentation
* integration test to ensure we can set the global logger
